torch/quantization/fuse_modules.py

```diff
@@ -137,6 +138,22 @@ def fuse_modules(model, modules_to_fuse, inplace=False, fuser_func=fuse_known_mo
     if not inplace:
         model = copy.deepcopy(model)
 
+    if modules_to_fuse is None:
```

Comment:
Can you add a test case showing how this extension works?
One problem with fusion is that we need a guarantee that the modules are called in sequence. There is no way to automate this unless one uses sequential or explicitly list the modules that are called in sequence.

Reply:
Yes, you are right. Without this guarantee from user, we cannot do the automatic fuse in eager mode. So, better drop this PR?

Reply:
Yes, please. Thanks!

Reply:
ok.